src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketRevisionAction.java

```diff
@@ -1,6 +1,5 @@
-package com.atlassian.bitbucket.jenkins.internal.status;
+package com.atlassian.bitbucket.jenkins.internal.scm;
```

Comment:
Does this change break deserialisation of the action in jobs saved using earlier versions of the plugin?

Reply:
Yes, I believe it will.

Reply:
Thanks, I guess it will have to keep living in the `status` package then